### ISSUE TITLE
JAWS is not reading collapsed state of Dropdown

### ISSUE THREAD

**Issue by keerthanameka25** [FreedomScientific/VFO-standards-support#445]
### Summary

> For Select Dropdown, I'm using (Alt + Down arrow) to view all the options  and selecting an option with ENTER. In IE browser, dropdown collapsed State is announced by JAWS but not in Chrome browser
> 
> Example:
> https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

### Expected result

> What did you expect to happen?

### JAWS should announce Dropdown collapsed.

> What actually did happen?

### JAWS is not announcing the collapsed state.

> Sample : https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

### Additional Information
Observed the issue in JAWS 2018 and Jaws 2020
OS : windows
browser: Chrome

**Comment by frex65** [comment]
Bug confirmed using JAWS 2020.2008.24 and Chrome 85.0.4183.121 . JAWS says "expanded" when a `<select>` box is opened, but not "collapsed" when it is closed.
